src/templates/base/2019/base_ebook.html

```diff
@@ -0,0 +1,170 @@
+{% extends "%s/2019/base.html" % lang %}
+<!-- TODO: I'm not sure what meta tags would be appropriate for a single-page ebook -->
```

Comment:
This HTML comment gets rendered to the client. I think this is the correct syntax for template-only comments.

```suggestion
{# TODO: I'm not sure what meta tags would be appropriate for a single-page ebook #}
```

Reply:
Removed this as no longer TODO anyway 😀